dashboard/src/components/AppView/DeploymentsTable/DeploymentsTable.tsx

```diff
@@ -0,0 +1,46 @@
+import * as React from "react";
+
+import LoadingWrapper from "../../../components/LoadingWrapper";
+import { IKubeItem, IResource } from "../../../shared/types";
+import isSomeResourceLoading from "../helpers";
+import DeploymentItem from "./DeploymentItem";
+
+interface IDeploymentTableProps {
+  deployments: Array<IKubeItem<IResource>>;
+}
+
+class DeploymentTable extends React.Component<IDeploymentTableProps> {
+  public render() {
+    const { deployments } = this.props;
+    let deploymentSection = <p>The current application does not contain any deployment.</p>;
+    if (deployments.length > 0) {
+      deploymentSection = (
+        <table>
+          <thead>
+            <tr>
+              <th>NAME</th>
+              <th>DESIRED</th>
+              <th>UP-TO-DATE</th>
+              <th>AVAILABLE</th>
+            </tr>
+          </thead>
+          <tbody>
+            {deployments.map(
+              d => d.item && <DeploymentItem key={d.item.metadata.name} deployment={d.item} />,
```

Comment:
too late but I'd have called it `resource` instead of `item` :(

Reply:
I'd avoid to keep adding changes to this PR, if you think that's needed please open a different issue.